File: sample/attr_accessor.rb
class Warrior
  attr_accessor :health
end

w = Warrior.new
w.health = 95
puts w.health

class Warrior
  def health
    @health
  end
  def health=(val)
    @health = val
  end
end

w = Warrior.new
w.health = 59
puts w.health

File: sample/AttrAccessor.java
class Warrior
{
  public void setHealth(int val)
  { 
    health = val; 
  }
  public int getHealth() 
  {
    return health; 
  }
  private int health;
}

public class AttrAccessor
{
    public static void main(String[] args) 
    {
      Warrior s = new Warrior();
      s.setHealth(95);
      System.out.println("Health = " + s.getHealth());
    }
}

